### ISSUE TITLE
[Feature - Horizontal] Implement Timeline-arrow as a way of connecting events

### ISSUE THREAD

**Issue by ReconVirus** [seanlowe/obsidian-timelines#44]
Integration with  timeline arrows,

Use case would be to "link" events together, or show a connections between them as a cause and effect 

https://github.com/javdome/timeline-arrows

a playtest demo of it working 

https://gistpreview.github.io/?b5a7a48e51d0b5d62aee718e10039e16

**Comment by seanlowe** [comment]
This is pretty cool. I'll add it to the to-do list

**Comment by seanlowe** [comment]
Looking into this, actually implementing the arrows should be fairly simple. I'm more wondering how I would be able to let the end user make those connections themselves. From the timeline-arrows README, each connection has to be have the ID of start and finish. 

Right now I generate an ID for each event from the normalized start date of an event, so perhaps I could add a field on events for `successor`, `arrow-to`, `points-to`, `next-event` or some such?

Events would probably look like:
```html
<div class="ob-timelines"
  data-title="First Event"
  data-start-date="2024-5-24"
  data-points-to="2024-6-11"
>
  Event Number One
</div>
```

```html
<div class="ob-timelines"
  data-title="Second Event"
  data-start-date="2024-6-11"
>
  Event Number Two
</div>
```

Does this seem an acceptable way to add this? What are your thoughts?

**Comment by ReconVirus** [comment]
Sorry for the long wait as im just seeing this now, didnt expect youll get to it this fast, and yeah actually that pretty much sums up how i figure it to work or a simple connects-to : (name of event) as a simpler way

**Comment by seanlowe** [comment]
Created an [issue](https://github.com/javdome/timeline-arrows/issues/10) on the timeline-arrows repo because I wasn't sure if groups were required or not in order to make this work. Turns out they are so I'm going to make it so that all items are automatically added to a single group in order for the arrows to render, I'm also going to implement the ability to add groups to your items as a whole for the timeline, which is issue #62. Both of these will now be added to the 2.4.0 milestone

**Comment by seanlowe** [comment]
This has been merged into the release/2.4.0 branch. Still need to implement #62 and do some QA before release will be ready. Getting there!
